assets/js/theme/global/lazysizes.js

```diff
@@ -0,0 +1,4 @@
+import 'lazysizes';
```

Comment:
If `lazysizes` module gets initialized automatically, you probably don't need this file. Just `import 'lazysizes` directly in `assets/js/theme/global.js`